adc/ade7913/ade7913.h

```diff
@@ -43,10 +43,15 @@ int ade7913_sample_regs_read(oid_t *device, int cs, ade7913_burst_reg_t *reg);
 int ade7913_lock(oid_t *device, int cs);
 int ade7913_unlock(oid_t *device, int cs);
 
+int ade7913_emi(oid_t *device, int cs, uint8_t val);
+
 int ade7913_enable(oid_t *device, int cs);
 int ade7913_disable(oid_t *device, int cs);
 
+int ade7913_temperature(oid_t *device, int cs);
+int ade7913_voltage(oid_t *device, int cs);
```

Comment:
`reset_soft` implementation was removed but declaration remained?

Reply:
Oh, I brought it back.